chips/nrf52/src/power.rs

```diff
@@ -0,0 +1,351 @@
+//! Power management
```

Comment:
A little more explaining this would be useful since I don't think it is intuitive why a power manager requires a USB callback.

Reply:
Added an explanation on the `Power` struct.